docs/user/tutorial/quickstart/java11.rst

```diff
@@ -0,0 +1,143 @@
+:Author: Torben Barsballe
+:Thanks: geotools-user list
+:Version: |release|
+:License: Creative Commons with attribution
+
+**********************
+ Java 11 Quickstart 
+**********************
+
+.. sectionauthor:: Torben Barsballe <tbarsballe@boundlessgeo.com>
+
+Welcome 
+=======
+
+GeoTools 21.0 and newer can be used in a Java 11 modular application. This requires a few additions on top of the regular tutorial application.
+
+This quickstart assumes you have already run through one of the regular quickstarts:
+
+ * `Eclipse <./eclipse.html>`_
+ * `NetBeans <./netbeans.html>`_
+ * `IntelliJ <./intellij.html>`_
+ * `Maven <./maven.html>`_
+
+Java Install
+============
+
+Instead of Java 8, we will want to install Java 11.
+
+#. Download the latest Java 11 JDK:
+
+   * Oracle JDK: http://www.oracle.com/technetwork/java/javase/downloads/
+   * OpenJDK: http://jdk.java.net/11/
```

Comment:
Consider putting the Oracle JDK in a information or warning box to explain it is not an open source download:

```
.. info:

  OpenJDK updates every six months, for longer term support consider:
  
  * Adopt OpenJDK: https://adoptopenjdk.net
  
  Oracle distributes a supported JDK 11 for their customers, which is available for development and testing:
  
  * http://www.oracle.com/technetwork/java/javase/downloads/
```

Reply:
Good idea, will add